### PR TITLE
Remove default annotations

### PR DESCRIPTION
**Pull request by Danielius1922**
Change default value of expiration in coap-gateway and
resource-aggregate for owner cache from 10m to 1m.

### CODE REVIEW

File: coap-gateway/service/config.go [hunk 1]
```diff
@@ -60,7 +60,7 @@ type COAPConfig struct {
 	Addr                     string                  `yaml:"address" json:"address"`
 	ExternalAddress          string                  `yaml:"externalAddress" json:"externalAddress"`
 	MaxMessageSize           int                     `yaml:"maxMessageSize" json:"maxMessageSize" default:"262144"`
-	OwnerCacheExpiration     time.Duration           `yaml:"ownerCacheExpiration" json:"ownerCacheExpiration" default:"10m"`
+	OwnerCacheExpiration     time.Duration           `yaml:"ownerCacheExpiration" json:"ownerCacheExpiration" default:"1m"`
```
Comment (jkralik) [inline coap-gateway/service/config.go]: default is not used by yaml parser - so you can remove it

File: resource-aggregate/service/config.go [hunk 2]
```diff
@@ -44,7 +44,7 @@ func (c *APIsConfig) Validate() error {
 }
 
 type GRPCConfig struct {
-	OwnerCacheExpiration time.Duration `yaml:"ownerCacheExpiration" json:"ownerCacheExpiration" default:"10m"`
+	OwnerCacheExpiration time.Duration `yaml:"ownerCacheExpiration" json:"ownerCacheExpiration" default:"1m"`
```
Comment (jkralik) [inline resource-aggregate/service/config.go]: default is not used by yaml parser - so you can remove it

Comment (Danielius1922) [reply resource-aggregate/service/config.go]: I see, I'll get rid of the defaults then.

Comment (Danielius1922) [reply resource-aggregate/service/config.go]: I've removed it from fields with `yaml` and `json` annotations, but kept it if `envconfig` was present (for example: 	https://github.com/plgd-dev/cloud/blob/v2/resource-directory/service/config.go#L63)

Comment (jkralik) [reply resource-aggregate/service/config.go]: It is not used anymore in any https://github.com/plgd-dev/cloud/blob/v2/<SERIVICE>/service/config.go.

Only https://github.com/plgd-dev/cloud/blob/v2/pkg/config pkg can contains env var for loading config path.

**Review by jkralik — COMMENTED** [review]
If you need you can clean up all services.

File: resource-aggregate/service/config.go (hunk 2 above)
Comment (Danielius1922) [reply resource-aggregate/service/config.go]: Removed from everything except c2c stuff, which can be removed only after the services are refactored.

**Review by jkralik — APPROVED** [review]

**Pull request merged**